### ISSUE TITLE
Crashes and Performance Issues under heavy raster loads / imports 

### ISSUE THREAD

**Issue by tushantin** [opentoonz/opentoonz#1713]
I'm using the Morevna build in Linux, but it may also occur in the standard builds in other OS, so please confirm. I have 16 gigs of ram and a decent processor and graphics card. 

I've noticed that I have performance issues in OT when I import large image files for backgrounds and such (usually over 5000px or 7000px width — or in my case, 7680 x 4320 resolution image), even if it's around 3 or 4 of them. I do some image-processing in the FX schematics on them, and hand-draw some animations in different levels. 

Even though my system resources don't reach a hundred percent (still enough ram, graphics and processing power available), OT seems to either lag when I switch between cells, or it crashes when I try to render with "use all cores" settings (sometimes even in "use half cores"; forcing me to only use a single one). 

I wonder if this is an issue, and if so, could be worked around / fixed in some way? Or is there some optimization feature that I might be missing? What can be done to make OT and its rendering more responsive for higher resolutions?

**Comment by artisteacher** [comment]
I've noticed the same lagginess - particularly with imported or loaded video. I've found if I merge the video frames onto a new raster level (with the same number of frames) and then remove the video file, the lagging significantly improves. Since the audio doesn't load with the video, it would be really helpful to have an option for video/psd images to automatically be converted to raster on import.
